server/src/main/clients/web-browser/src/libs/projects/ProjectDashboard/ProjectDashboard.tsx

```diff
@@ -0,0 +1,22 @@
+import './ProjectDashboard.scss';
+import GaugeChart from 'react-gauge-chart';
+
+function ProjectDashboard() {
+  return (
+    <section>
+      <h2>Post Tracking</h2>
+      <GaugeChart
+        id="gauge-chart1"
+        nrOfLevels={5}
+        hideText
+        needleBaseColor="#00000000"
```

Comment:
Is there is flag to hide the needle rather than setting the alpha to 0? Similar to hideText?

Reply:
I didn't see one, also I'm not sure how much I like this library so I might want to replace it later, it was the closest one I could find